lib/galaxy/datatypes/infernal.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+
+import logging
+import os
+
+from galaxy.datatypes.data import Text
+from galaxy.datatypes.data import get_file_peek
+
+from galaxy.datatypes.metadata import MetadataElement
+
+log = logging.getLogger(__name__)
+
+
+class Infernal_CM_1_1(Text):
+    file_ext = "cm"
+
+    MetadataElement(name="number_of_models", default=0, desc="Number of covariance models", readonly=True, visible=True, optional=True, no_value=0)

```

Comment:
We don't calculate the number of models since the removal of grep, so we can remove it I guess and the import as well.